Source: http://lxde.svn.sourceforge.net/viewvc/lxde/trunk/lxrandr/configure.in?r1=1673&r2=1822
Upstream: Yes
Reason: fix the problem on runs configure in infinite loop.
--- lxrandr/configure.in  2009/06/24 20:59:58 1673
+++ lxrandr/configure.in  2009/07/15 12:11:10 1822
@@ -5,6 +5,9 @@
 AM_CONFIG_HEADER(config.h)
 AM_MAINTAINER_MODE
 
+# Init intltool
+IT_PROG_INTLTOOL
+
 AC_ISC_POSIX
 AC_PROG_CC
 AM_PROG_CC_STDC

